<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="drawingArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1050</width>
      <height>768</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>20</y>
      <width>15</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>60</y>
      <width>15</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cordinateX">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>20</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cordinateY">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>60</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mSetPointButton">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Point</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mXRelfectButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>130</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>About X Axis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mYReflectButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>170</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>About Y Axis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mXYReflectButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>210</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>About X=Y Axis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mRotateFunctionButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>250</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate About Point</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mRotateButton">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>450</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="arbitaryY">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>370</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="arbitaryX">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>330</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>330</y>
      <width>15</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>370</y>
      <width>15</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>350</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="angleEditText">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>350</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mDrawButton">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>60</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
